api/v1alpha1/alluxioruntime_types.go

```diff
@@ -154,6 +154,10 @@ type AlluxioFuseSpec struct {
 	// PodMetadata defines labels and annotations that will be propagated to Alluxio's fuse pods
 	// +optional
 	PodMetadata PodMetadata `json:"podMetadata,omitempty"`
+
+	// Use the host's pid namespace, default false.
+	// +optional
+	HostPID bool `json:"hostPID,omitempty"`
```

Comment:
In case we will disable this configuration in future, we can set this by using environment variables or annotations. WDYT?

Reply:
agree, will use the  environment variables.